$FreeBSD$

Replace the defaults for GTK styles with more GTK-style defaults and not Windows-style.

--- stc-styles.rc.cfg.orig	2012-08-12 23:05:56 UTC
+++ stc-styles.rc.cfg
@@ -1,5 +1,5 @@
 common.defs.msw={"helv":"Verdana","mono":"Courier New","lnsize":8,"other":"Comic Sans MS","backcol":"#FFFFFF","size":10}
-common.defs.gtk={"mono":"Courier New","helv":"Verdana","lucd":"Verdana","other":"Comic Sans MS","size":8,"lnsize":6,"backcol":"#FFFFFF"}
+common.defs.gtk={"mono":"Monospace","helv":"Sans","lucd":"Sans","other":"Serif","size":10,"lnsize":8,"backcol":"#FFFFFF"}
 common.styleidnames = {wxSTC_STYLE_DEFAULT: "Style default", wxSTC_STYLE_LINENUMBER: "Line numbers", wxSTC_STYLE_BRACELIGHT: "Matched braces", wxSTC_STYLE_BRACEBAD: "Unmatched brace", wxSTC_STYLE_CONTROLCHAR: "Control characters", wxSTC_STYLE_INDENTGUIDE: "Indent guide"}
 [style.python]
 setting.python.-3=
